pkg/proxy/iptables/proxier.go

```diff
@@ -920,6 +920,11 @@ func (proxier *Proxier) syncProxyRules() {
 			writeLine(proxier.natChains, utiliptables.MakeChainLine(chainName))
 		}
 	}
+	// ensure KUBE-MARK-DROP chain exist but not do any change
+	if _, err := proxier.iptables.EnsureChain(utiliptables.TableNAT, KubeMarkDropChain); err != nil {
+		klog.Errorf("Failed to ensure that %s chain %s exists: %v", utiliptables.TableNAT, KubeMarkDropChain, err)
+		return
```

Comment:
in a dual stack cluster,  that we know this rule does not exist in one of the ip families, 
kube-proxy will stop to work in a one family or will stop to work at all?

Reply:
Sorry I didn't notice this before. Then both ipvs and iptbales mode have this issue, I think one of the ip familier don't have this rule is a bug, maybe we can wait https://github.com/kubernetes/kubernetes/pull/94474 to be merged.

Reply:
Are we still waiting on https://github.com/kubernetes/kubernetes/pull/94474 for this? 

It seems like there are two separate issues here. For IPVS specifically, we are doing what iptables used to do before https://github.com/kubernetes/kubernetes/pull/85527. Sounds like we should fix that first independent of the dual-stack PR? Am I missing something there?